src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Constants/ApplicationInsightsScopeKeys.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host.Loggers
```

Comment:
Can you check/fix all the namespaces? I think you want to use the default namespace for your package: Microsoft.Azure.WebJobs.Logging.ApplicationInsights